sklearn/tree/_classes.py

```diff
@@ -897,8 +908,7 @@ class in a leaf.
 
         Returns
         -------
-        p : array of shape (n_samples, n_classes), or a list of n_outputs
-            such arrays if n_outputs > 1.
+        proba : array of shape (n_samples, n_classes), or a list of such arrays
```

Comment:
The previous information was still relevant. When dtype information is longer that 1 line, it can be continued with `\`:
```
        proba : array of shape (n_samples, n_classes), or a list of n_outputs \
            such arrays if n_outputs > 1.
```